test/plugins/tokens.test.js

```diff
@@ -218,24 +225,119 @@ describe('token plugin test', () => {
         });
     });
 
-    describe('DELETE /tokens/{id}', () => {
+    describe('PUT /tokens/{id}', () => {
         let options;
-        const tokenId = testToken.id;
 
         beforeEach(() => {
             options = {
-                method: 'DELETE',
+                method: 'PUT',
                 url: `/tokens/${tokenId}`,
                 credentials: {
                     username,
                     scope: ['user']
+                },
+                payload: {
+                    name: testToken.name,
+                    description: 'a new description'
                 }
             };
 
             tokenFactoryMock.get.resolves(tokenMock);
+            userMock.tokens = tokensGetterMock([tokenMock]);
+            userFactoryMock.get.withArgs({ username }).resolves(userMock);
```

Comment:
Redundant of https://github.com/screwdriver-cd/screwdriver/pull/573/files#diff-ee5dae69c9de402833a459d19c780d18R74